docs/local.sh

```diff
@@ -28,9 +28,9 @@ openBrowser
 # Otherwise it'll crash hugo due to missing files.
 # You can find them in .dockerignore
 
-docker run -t --rm \
+docker run -t -i --rm \
```

Comment:
to fix Ctrl + C handling

Reply:
i thought `-t` already fixed it;

Reply:
Me too
https://docs.docker.com/engine/reference/run/#foreground